tests/oauth_token_fabricator.go

```diff
@@ -0,0 +1,34 @@
+package tests
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-oauth2/oauth2/v4/models"
+
+	"google-scraper/services/oauth"
+
+	"github.com/go-oauth2/oauth2/v4"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+)
+
+func FabricateOAuthToken(client *models.Client) oauth2.TokenInfo {
+	info := &models.Token{
+		ClientID:        client.GetID(),
+		Access:          uuid.New().String(),
+		AccessCreateAt:  time.Now(),
```

Comment:
@junan Should we use `time.Now().Local()` ?
```suggestion
		AccessCreateAt:  time.Now().Local(),
```